### DISCUSSION TITLE
Combining refiner and detailer

### DISCUSSION THREAD

**Discussion by omeran80** [vladmandic/sdnext#3566] [Ideas]
I've been using ADetailer up till now, and it stopped working, so I thought I'd give the built-in detailer a try. I understood that most of the settings are the refiner tab, but that tab doesn't even appear in image-to-image.
Perhaps it's possible to combine the two into a single tab?

**Comment by vladmandic** [comment]
i'll add refiner section to img2img, it has value regardless of detailer.
btw, in the future this would be better suited as feature request.

**Comment by omeran80** [comment]
(thanks, will, do next time, I'm still a bit new at this)
Would that also allow setting different resolution for the detailer? because it seems that now it's stuck on 1024x1024 no matter what I use in the refiner section.

**Reply by vladmandic** [reply]
yes

**Reply by vladmandic** [reply]
ok, this is now live in dev branch.

**Reply by omeran80** [reply]
Great, thanks!
